### ISSUE TITLE
util.id appears unused

### ISSUE THREAD

**Issue by domoritz** [vega/vega#3158]
https://github.com/vega/vega/blob/9a312540582b4d5dd25439029184b0e51e6b88cb/packages/vega-util/src/accessors.js#L4

Maybe I am missing something but I only see it being used in the tests.

**Comment by hydrosquall** [comment]
Making sure I understand the suggestion:

If the util is unused, we might consider deleting it or moving it to a "test utils" folder?

**Comment by domoritz** [comment]
Yeah, that sounds like something I thought about when I filed this.
